lib/src/collect.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library coverage.collect;
+
+import 'dart:async';
+
+import 'devtools.dart';
+import 'util.dart';
+
+const _retryInterval = const Duration(milliseconds: 200);
+
+Future<Map> collect(String host, int port, bool resume, bool waitPaused,
```

Comment:
Would be great if you could expose `collect` as public to be used by other packages. This allows to collect coverage in-process just with a method call (without launching another Dart process for collect_coveralls and reading stdout). I can use it this way but it is against the guidelines to import from otherpackage/src.